win32ss/user/ntuser/callback.c

```diff
@@ -333,6 +334,7 @@ co_IntCallWindowProc(WNDPROC Proc,
    Arguments->wParam = wParam;
    Arguments->lParam = lParam;
    Arguments->lParamBufferSize = lParamBufferSize;
+   Arguments->Result = 0;
```

Comment:
Result isn't the only uninitialized thing here -- on x64 this structure is also going to have a bunch of padding in it. So it would be best to `RtlZeroMemory(Arguments, sizeof(*Arguments));` after the if (i.e. in both the stack and the pool case)